controllers/get-reviews-comments-controllers.js

```diff
@@ -0,0 +1,19 @@
+const reviews = require('../db/data/test-data/reviews');
+  const comments = require('../db/data/test-data/comments');
+  const {fetchReviewsByID} = require('../models/get-reviews-id-models')
+  const {fetchReviewCommentsByID} = require('../models/get-reviews-comments-models');
+  
+  function getReviewCommentsByID(request, response, next){
+      const {review_id} = request.params;
+      const promiseForComment = [fetchReviewCommentsByID(review_id)];
+  
+      if(review_id){promiseForComment.push(fetchReviewsByID(review_id))};
```

Comment:
I understand this if statement, but you're also going to run fetchReviewCommentsByID if there isn't a valid review_id, so not sure it's completely needed here.